### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by LittleToonCat** [open-toontown/open-toontown#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 2Epik4u** [comment]
Will there be lawbot field offices in open toontown? since it almost completed but never was added to the game
